ansible/playbook.yml

```diff
@@ -9,3 +9,4 @@
     - php
     - xdebug
     - app
+    - composer
```

Comment:
I think there is no need to add new role, we can just add git to `server.packages` list, and it will be installed